ms/MSOper/MSMetaData.cc

```diff
@@ -1428,6 +1428,13 @@ std::map<ScanKey, std::set<uInt> > MSMetaData::getScanToSpwsMap() const {
     return scanToSpwMap;
 }
 
+vector<std::set<ScanKey> > MSMetaData::getSpwToScansMap() const {
```

Comment:
Yes. Values of spw are simply the row numbers in the SPECTRAL_WINDOW subtable, so a vector seemed like the most natural choice.